src/clojure/nrepl/server.clj

```diff
@@ -133,11 +135,58 @@
   java.io.Closeable
   (close [this] (stop-server this)))
 
+(defn inet-socket [bind port]
```

Comment:
I'd deliniate with some section comments the boundaries of the UNIX socket part of the implementation.